db/fixtures/miq_product_features.yml

```diff
@@ -5095,6 +5095,11 @@
       :description: View Ansible Tower Job Templates
       :feature_type: view
       :identifier: configuration_script_view
+      :children:
+      - :name: List
+        :description: List Configuration Script Sources
+        :feature_type: view
+        :identifier: configuration_script_sources_list
```

Comment:
@h-kataria would appreciate your 👀 - should I wait for #13526 ? I am not sure it makes sense to have it ansible specific after reviewing that, would something like this work?